Dockerfile

```diff
@@ -29,16 +30,21 @@ CMD go vet ./... && go test ./...
 FROM base as development
 
 RUN ["go", "install", "github.com/githubnemo/CompileDaemon@latest"]
-EXPOSE 8001
+EXPOSE 9008
 
 CMD CompileDaemon -log-prefix=false -build="go build ./cmd/server" -command="./server"
 
 
 # Productive build.
 FROM alpine:3.13.2
 WORKDIR /root/
+RUN apk add bash
 
 COPY --from=builder /root/server .
-EXPOSE 8001
+COPY --from=builder /root/manage .
+COPY entrypoint .
```

Comment:
Let's use a CheckServer command so need not slow down the start of this service,